insights/parsers/systemd/config.py

```diff
@@ -135,9 +138,9 @@ def parse_systemd_ini(content):
         dict_section = {}
         for option in Config.options(section):
             value = Config.get(section, option).splitlines()
-            value = map(lambda s: s.rstrip("\\").strip(), value)
-            value = filter(lambda s: bool(s), value)
-            dict_section[option] = "\n".join(value)
+            value = list(unsplit_lines(value))
+            """ If the len of value is 1, then set the value as string"""
+            dict_section[option] = value[0] if len(value) == 1 else value
```

Comment:
This line should be `dict_section[option] = value[0] if len(value) == 1 else value if len(value) > 1 else ''`
That's to say for value is empty, just assign it as ''